docs/Overview.md

```diff
@@ -0,0 +1,291 @@
+| `onSuccess`        | `(response: CostCalculatorEstimateResponse) => void` | No       | Callback with the successful estimation data                |
+| `onError`          | `(error: Error) => void`                             | No       | Error handling callback                                     |
+
+#### estimationParams Properties
+
+| Property                | Type      | Description                                                  |
+| ----------------------- | --------- | ------------------------------------------------------------ |
+| `title`                 | `string`  | Custom title for the estimation report                       |
+| `includeBenefits`       | `boolean` | When true, includes benefits information in the response     |
+| `includeCostBreakdowns` | `boolean` | When true, includes detailed cost breakdowns in the response |
+
+#### defaultValues Properties
+
+| Property            | Type     | Description                 |
+| ------------------- | -------- | --------------------------- |
+| `countryRegionSlug` | `string` | Pre-selected country/region |
+| `currencySlug`      | `string` | Pre-selected currency       |
+| `salary`            | `string` | Pre-filled salary amount    |
+
+### CostCalculatorResults
+
+A component to display cost calculation results.
+
+| Prop             | Type                                 | Required | Description                    |
+| ---------------- | ------------------------------------ | -------- | ------------------------------ |
+| `employmentData` | `CostCalculatorEstimateResponseData` | Yes      | The estimation data to display |
+
+## Authentication
+
+You need to implement a server endpoint to securely handle authentication with Remote. This prevents exposing client credentials in your frontend code.
+
+To do this you can follow the code in the [server.js][https://github.com/remoteoss/remote-flows/blob/main/example/server.js#L8]
+
+### API Gateway Endpoints
+
+- **Development/Testing**: `https://gateway.partners.remote-sandbox.com`
+- **Production**: `https://gateway.remote.com/`
+
+## Styling Options
+
+### Using Default Styles
+
+Import the CSS file in your application:
+
+```css
+@import '@remoteoss/remote-flows/index.css';
+```
+
+### Theme Customization
+
+```tsx
+<RemoteFlows
+  theme={{
+    primaryBackground: '#ffffff',
+    primaryForeground: '#364452',
+    accentBackground: '#e3e9ef',
+    accentForeground: '#0f1419',
+    danger: '#d92020',
+    focus: 'blue',
+    borderInput: '#cccccc',
+  }}
+>
+  {/* Your components */}
+</RemoteFlows>
+```
+
+| **Token**           | **Description**                   |
+| ------------------- | --------------------------------- |
+| `focus`             | Focus color.                      |
+| `borderInput`       | Border color for input fields.    |
+| `primaryBackground` | Primary background.               |
+| `primaryForeground` | Primary foreground.               |
+| `accentBackground`  | Accent background.                |
```

Comment:
We should be more explicit about what changing accent means. For example, it's used in the select component to change the border color when the component is focused.